src/users/users.service.ts

```diff
@@ -30,4 +30,12 @@ export class UsersService {
       throw new InternalServerErrorException(error.message);
     }
   }
+
+  /**
+   *
+   * grabs all users with the role admin and returns them as an array of Users
```

Comment:
🔧 This isn't a great purpose statement, it more or less restates the signature and the name of the function. How about:

```suggestion
   * Finds all users with the admin role.
```

Reply:
I'll try and write better purpose statements next time